projects/dac_fmc_ebz/Readme.md

```diff
@@ -0,0 +1,8 @@
+# DAC-FMC-EBZ HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/eval-ad9172)
+  * Parts : [Dual, 16-Bit, 12.6 GSPS RF DAC with Channelizers](https://www.analog.com/ad9172)
```

Comment:
This project supports multiple parts. 

See the list in the user guide https://wiki.analog.com/resources/eval/user-guides/ad-dac-fmc-ebz 
or in the  project config file : https://github.com/analogdevicesinc/hdl/blob/master/projects/dac_fmc_ebz/common/config.tcl

Reply:
I re-read your comment, and looked at the wiki page - and fixed things...